Sloth.Core/Jobs/CybersourceBankReconcileJob.cs

```diff
@@ -42,34 +41,36 @@ public CybersourceBankReconcileJob(SlothDbContext context, ICyberSourceBankRecon
                 if (!integrations.Any())
                 {
                     log.Information("Early exit, no active integrations found.");
-                    yield break;
```

Comment:
You don't need this?

Reply:
Not anymore. The `CyberSourceBankReconcileService` now does the work of saving the blob records. So they don't need to be passed up to let the caller save them.